src/components/Spinner.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
 const Spinner = ({ className }) => (<div className={`Spinner ${className}`} />)
 
+Spinner.PropTypes = {
```

Comment:
`PropTypes` -> `propTypes`. In addition, this component takes one prop -- className, which is a string.